tools/integration_tests/util/client/control_client.go

```diff
@@ -19,13 +19,27 @@ import (
 	"fmt"
 	"log"
 	"strings"
+	"time"
 
 	control "cloud.google.com/go/storage/control/apiv2"
 	"cloud.google.com/go/storage/control/apiv2/controlpb"
+	"github.com/googlecloudplatform/gcsfuse/v2/internal/storage/storageutil"
 )
 
 func CreateControlClient(ctx context.Context) (client *control.StorageControlClient, err error) {
 	client, err = control.NewStorageControlClient(ctx)
+	clientConfig := &storageutil.StorageClientConfig{MaxRetrySleep: 30 * time.Second, RetryMultiplier: 2}
+
+	client.CallOptions.RenameFolder = storageutil.StorageControlClientRetryOptions(clientConfig)
+	client.CallOptions.GetFolder = storageutil.StorageControlClientRetryOptions(clientConfig)
+	client.CallOptions.GetStorageLayout = storageutil.StorageControlClientRetryOptions(clientConfig)
+	client.CallOptions.CreateFolder = storageutil.StorageControlClientRetryOptions(clientConfig)
+	client.CallOptions.DeleteFolder = storageutil.StorageControlClientRetryOptions(clientConfig)
+	client.CallOptions.CreateManagedFolder = storageutil.StorageControlClientRetryOptions(clientConfig)
```

Comment:
The managed folder retry options also should come from the application logic itself.

Reply:
This is not part of application logic and only used in creating test data on bucket to perform e2e tests.